Author: Apollon Oikonomopoulos <apoikos@debian.org>
Description: Override local loading path
 Use the package's /usr/share/puppet/locale instead of /usr/share/locale.
Bug-Debian: https://bugs.debian.org/866972
--- a/lib/puppet/gettext/config.rb
+++ b/lib/puppet/gettext/config.rb
@@ -3,7 +3,7 @@
 
 module Puppet::GettextConfig
   LOCAL_PATH = File.absolute_path('../../../locales', File.dirname(__FILE__))
-  POSIX_PATH = File.absolute_path('../../../../../share/locale', File.dirname(__FILE__))
+  POSIX_PATH = '/usr/share/puppet/locale'
   WINDOWS_PATH = File.absolute_path('../../../../../../../puppet/share/locale', File.dirname(__FILE__))
 
   # This is the only domain name that won't be a symbol, making it unique from environments.
